src/pages/companies-list/companies-list.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
Looking to other components, the test file is under a `__test__` folder, can we keep this pattern?